packages/rpc/src/chainStorageWatcher.ts

```diff
@@ -12,8 +12,8 @@ type Subscriber<T> = {
 
 const defaults = {
   newPathQueryDelayMs: 20,
-  refreshLowerBoundMs: 2000,
-  refreshUpperBoundMs: 4000,
+  refreshLowerBoundMs: 4000,
```

Comment:
is Product good with this doubling of minimal refresh time?

Reply:
Good question, I think since these are just defaults and configurable, erring on the side of less spam would be safer, but we can always pass in smaller values for certain queries in certain dapps